jobs/config.json

```diff
@@ -12381,6 +12399,25 @@
       "sig-scheduling"
     ]
   },
+  "pull-kubernetes-e2e-gce-dind-ipv6": {
+    "args": [
+      "--cluster=",
+      "--deployment=gce-dind",
+      "--gce-dind-image=rpothier/kubeadm-dind-cluster:k8s-1.10",
```

Comment:
Does it mean the latest k8s build from the 1.10 branch? Is it possible to maintain the image on gcr.io?

Reply:
This refers to the branch of kubeadm-dind-cluster. This image will be replaced by a stable branch of Mirantis/kubeadm-dind-cluster when it is available. See https://github.com/Mirantis/kubeadm-dind-cluster/issues/115

Reply:
> Once the Mirantis stable branch is created, this --gce-dind-image argument will go away

Thanks for the explanation (https://github.com/Mirantis/kubeadm-dind-cluster/issues/115 is helpful). Seems fine to keep this for now.